server/src/ad-banner.js

```diff
@@ -25,9 +29,12 @@ exports.AdBanner = class AdBanner extends React.Component {
           </p>
         </Localized>;
     } else if (this.props.isOwner && !this.props.hasFxa) {
```

Comment:
With #4959 fix, condition here has changed to 

`} else if (!this.props.hasFxa) {`